requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 wdb
 ipdb
 Flask-DebugToolbar
+git+git://github.com/reanahub/reana-workflow-commons.git#egg=reana-workflow-commons
```

Comment:
shall we also add the requirement to `setup.py`?

Reply:
Yep, thanks